internal/cluster/finalizing_stages.go

```diff
@@ -40,6 +40,13 @@ var finalizingStages = []models.FinalizingStage{
 	models.FinalizingStageDone,
 }
 
+var nonFailingFinalizingStages = []models.FinalizingStage{
+	models.FinalizingStageWaitingForOlmOperators,
```

Comment:
Do we need this stage? It seems like 
https://github.com/openshift/assisted-installer/pull/779/files#diff-1da579fda3bc689940661d48dd744a46f23824270427207cfae4f8c7448eddd2R552
it is not really stage, this is just retry on getting operators from service? I think we can start with FinalizingStageWaitingForOlmOperatorsCsvInitialization

Reply:
I will unify with the initialization